src/library/scala/collection/immutable/NumericRange.scala

```diff
@@ -383,15 +440,19 @@ object NumericRange {
           if (step == num.fromInt(stepint)) {
             return {
               if (isInclusive) Range.inclusive(startint, endint, stepint).length
-              else             Range          (startint, endint, stepint).length
+              else Range(startint, endint, stepint).length
             }
           }
         }
       }
       // If we reach this point, deferring to Int failed.
       // Numbers may be big.
       if (num.isInstanceOf[Numeric.BigDecimalAsIfIntegral]) {
-        bigDecimalCheckUnderflow(start, end, step)  // Throw exception if math is inaccurate (including no progress at all)
+        bigDecimalCheckUnderflow(
```

Comment:
Much easier to verify arguments if they're on the same line!  Don't wrap to one-line-per-arg just because there's a comment on the line :(